### PR TITLE
Speaker language API request 

### PR DESCRIPTION
**Pull request by martinavagyan**
Since we have two user groups testing the applications one in French the other in Dutch, this fix is necessary to give better usability.

- Get the user language #101
- Progress bar look #100

### CODE REVIEW

File: src/zeeguu_exercises/static/js/app/exercises/exercise.js [hunk 1]
```diff
@@ -83,6 +84,7 @@ Exercise.prototype = {
 		this.size  = size;
 		this.shake = new ShakeAnimation();
 		this.exFeedback = new Feedback(this.resultSubmitSource,this.session);
+        Session.getLanguage((text)=>{this.lang = text});//Set the language with callback
```
Comment (mircealungu) [inline src/zeeguu_exercises/static/js/app/exercises/exercise.js]: great idea to address this! 
i was going to say that the accent was wrong in one of the exercises :) 

however, don't use the learned_language but rather, take the language
from the bookmark (i think it should have a from_lang and a to_lang)

this way it sounds right, also for people who have multiple languages 
in their profile!

**Comment by martinavagyan** [comment]
@mircealungu  the requested changes are now implemented. That was a good point.

File: src/zeeguu_exercises/static/js/app/speaker.js [hunk 2]
```diff
@@ -5,12 +5,22 @@ export default class Speaker {
 
     /**
      * @param {String} txt - the text to be read.
-     * @param {String} lang - speak the text in language. Defualt english.
+     * @param {String} lang - red the text in language. @example 'nl-NL'
```
Comment (mircealungu) [inline src/zeeguu_exercises/static/js/app/speaker.js]: red? read?

why is it nl-NL?

Comment (martinavagyan) [inline src/zeeguu_exercises/static/js/app/speaker.js]: That is how their API example worked, but I have just tested, and it works with just 'nl' as well. The new commit fixes this.

**Pull request merged**